glnx-errors.h

```diff
@@ -109,7 +109,8 @@ typedef struct {
 static inline void
 glnx_cleanup_auto_prefix_error (GLnxAutoErrorPrefix *prefix)
 {
-  g_prefix_error (prefix->error, "%s: ", prefix->prefix);
+  if (prefix->error && *(prefix->error))
```

Comment:
I think we should have a tag for these kinds of optimizations; those that don't actually make any practical difference in the big picture, but give the author/reviewer these warm fuzzy feelings knowing that something runs ever so slightly faster thanks to their insights, that something that used to take N CPU cycles, now takes N-5 cycles, where N > 100000000. A meditative relaxation exercise of sorts, allowing software engineers to commune with the craft they have come to learn and love. I'll call them "cozy patches".

This is a cozy patch. I'd say [this](https://github.com/projectatomic/rpm-ostree/pull/852#discussion_r124105791) is an even better cozy patch (the greater the micro-gains, the cozier it is). Monetarily 💰, it's not the most profitable way to spend one's time (or one's employer's time), but it's a great pick-me-up at the end of the day or when you need a breath of fresh air ☁️ between sessions of intense code-staring. 🖥 

I think my golden rule for cozy patches is that they should always be considered optional and never actively steal time 🕙 away from other, more serious, work. If they take more than 5 minutes to write and review (combined), then you've gone over. Any time more than that, it has to actually have a tangible performance benefit.

From now on, if I say "here is a cozy patch", then you'll know what I mean and apply the level of attention and urgency that it (doesn't) deserve(s). :)